### PR TITLE
[OrderBundle] Unification of service names

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
|-----------------|-----
| Branch?         | 1.14
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | yes
| Related tickets | #17093
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Resources/config/services.xml [hunk 1]
```diff
@@ -237,6 +237,7 @@
             <argument type="service" id="sylius.factory.order_sequence" />
             <argument type="service" id="sylius.manager.order_sequence" />
         </service>
+        <service id="sylius.number_generator.sequential_order" alias="sylius.sequential_order_number_generator" />
```
Comment (Wojdylak) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services.xml]: Consider this: is it set up correctly? The service with the same ID is in the OrderBundle, so I added the same alias in both places.

Comment (GSadee) [reply src/Sylius/Bundle/CoreBundle/Resources/config/services.xml]: I guess the one in OrderBundle should be enough

File: UPGRADE-1.14.md [hunk 2]
```diff
@@ -35,16 +35,23 @@
...
+    | `Sylius\Bundle\OrderBundle\Console\Command\RemoveExpiredCartsCommand` | `sylius.console.command.remove_expired_carts`                                       |
+    | `sylius.order_modifier`                                               | `sylius.modifier.order`                                                             |
+    | `sylius.order_item_quantity_modifier`                                 | `sylius.modifier.order_item_quantity`                                               |
+    | `sylius.order_number_assigner`                                        | `sylius.number_assigner.order_number`                                               |
+    | `sylius.adjustments_aggregator`                                       | `sylius.aggregator.adjustments_by_label`                                            |
+    | `sylius.expired_carts_remover`                                        | `sylius.remover.expired_carts`                                                      |
+    | `sylius.sequential_order_number_generator`                            | `sylius.number_generator.sequential_order`                                          |
```
Comment (GSadee) [inline UPGRADE-1.14.md]: I wonder if we shouldn't split this table per bundle 🤔

**Review by GSadee — DISMISSED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Karol! :1st_place_medal:

**Pull request merged**